src/adhocracy_core/adhocracy_core/websockets/server.py

```diff
@@ -51,6 +51,13 @@ def subscribe(self, client: Hashable, resource: IResource) -> bool:
         if self.is_subscribed(client, resource):
             return False
         path = resource_path(resource)
+
+        # ensure that no private info leaks to the outside
+        if path.startswith('/principals/') or path.startswith('/catalogs/'):
+            logger.info('Unallowed subscribtion')
+            raise WebSocketError('forbidden', 'You are not allowed to '
+                                 'subscribe on {}'.format(path))
+
```

Comment:
> can we merge it nonethless as a quickfix and you implement the proper solution soon?

We already have a quick workaround (disable websockets), so we don't have to bypass our definition of done for this fix.

Do you have time to work on this or  should  I do it?